main.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
```

Comment:
[OPTIONAL] It would be nicer to remove the commented code to keep your code clean.